Formula/lexbor.rb

```diff
@@ -0,0 +1,126 @@
+class Lexbor < Formula
+  desc "Fast embeddable web browser engine written in C with no dependencies"
+  homepage "https://lexbor.com/"
+  url "https://github.com/lexbor/lexbor/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "e9ba3db87b077cde8ee189eb0851e8b11823c778c6844adcce537357dcca8ae1"
+  license "Apache-2.0"
+
+  depends_on "cmake" => [:build, :test]
+
+  resource("testdata") do
+    url "https://github.com/lexbor/lexbor/archive/refs/tags/v1.0.0.tar.gz"
+    sha256 "e9ba3db87b077cde8ee189eb0851e8b11823c778c6844adcce537357dcca8ae1"
+  end
```

Comment:
The fact that the formula doesn't give you access to the source tarball in the `test` block is intentional: you're not supposed to do this.

It would be better if you extracted a test case from their test suite and implement it in the `test` block. Or, even better, simulate one way a user might actually use lexbor.

Reply:
Another option, if you must use the data, is to copy the necessary items out of the unpacked tarball at install time. This can be stored in `pkgshare`. This eliminates the need for an additional download at test time.

Reply:
Thank you @carlocab and @alebcay for your help 👍 I putted test data and examples to the `pkgshare` directory.
Addressed

Reply:
Sorry, but the formula is still unmaintainable as it stands. We really don't need a specialised test method for one formula, nor do we want a `CMakeLists.txt` consisting of more than ten lines declared in the formula.

I think you should follow my earlier suggestion:

> It would be better if you extracted a test case from their test suite and implement it in the `test` block. Or, even better, simulate one way a user might actually use lexbor.

Reply:
@carlocab Ok, I have tried. I updated the formula based on the [tinyxml2](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/tinyxml2.rb) example.
Addressed